src/module.ml

```diff
@@ -186,3 +203,11 @@ let wrapped_compat t =
 module Name_map = struct
   type nonrec t = t Name.Map.t
 end
+
+let is_public t =
```

Comment:
The pattern matching part should be in `Visibility.is_public`

Reply:
will fix